Tests/TurfTests/PolygonTests.swift

```diff
@@ -142,6 +142,153 @@ class PolygonTests: XCTestCase {
 
         XCTAssertEqual(expectedDiameter, diameter, accuracy: 0.25)
     }
+    
+    func testPolygonCentre() {
+        // Adopted from https://github.com/Turfjs/turf/blob/3b20c568e5638f680cde39c26b56fbcf034133f2/packages/turf-center/test.js
+        let coordinate = LocationCoordinate2D(latitude: 45.7536760235992, longitude: 4.841880798339844)
```

Comment:
I suppose the Lyon example would be more offset than expected when using `mid(_:_:)` because of a longstanding issue in which `LocationCoordinate2D.distance(to:)` uses a different approximation of the radius of the Earth than Turf.js does these days: #26.